core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -169,6 +170,37 @@ struct SpaceAwareAccessor<AnonymousSpace, NestedAccessor> {
   friend struct SpaceAwareAccessor;
 };
 
+// Like atomic_accessor_relaxed proposed for ISO C++26 but with
+// defaulted memory scope - similar to how desul's AtomicRef has a memory scope
+template <class ElementType, class MemoryScope = desul::MemoryScopeDevice>
+struct AtomicAccessorRelaxed {
+  using element_type = ElementType;
+  using reference =
+      desul::AtomicRef<ElementType, desul::MemoryOrderRelaxed, MemoryScope>;
+  using data_handle_type = ElementType*;
+  using offset_policy    = AtomicAccessorRelaxed;
+
+  static_assert(
+      std::is_same_v<std::remove_cv_t<element_type>, element_type>,
+      "AtomicAccessorRelaxed can only be used for non-const element types");
+
+  KOKKOS_FUNCTION
+  explicit operator default_accessor<element_type>() const {
+    return default_accessor<element_type>{};
+  }
+
+  KOKKOS_FUNCTION
+  reference access(data_handle_type p, size_t i) const noexcept {
+    return reference(p[i]);
+  }
+
+  KOKKOS_FUNCTION
+  constexpr data_handle_type offset(data_handle_type p, size_t i) const
```

Comment:
You meant the `constexpr` here or you missed it?

Reply:
I removed it.